packages/@aws-cdk/app-staging-synthesizer-alpha/test/app-staging-synthesizer.test.ts

```diff
@@ -16,7 +16,7 @@ describe(AppStagingSynthesizer, () => {
 
   beforeEach(() => {
     app = new App({
-      defaultStackSynthesizer: AppStagingSynthesizer.defaultResources({ appId: APP_ID }),
+      defaultStackSynthesizer: AppStagingSynthesizer.defaultResources({ appId: APP_ID, stagingBucketEncryption: BucketEncryption.S3_MANAGED }),
```

Comment:
For most of these tests, I used `S3_MANAGED`, since we intend to make this the default in the future.